docs/permissionless-node-setup.md

```diff
@@ -0,0 +1,127 @@
+# Permissionless node setup
+
+This guide describes how to run a permissionless node that:
+
+- Synchronizes the network
+- Expose a JSON RPC interface, acting as an archive node
+
+Note that sequencing and proving functionalities are not covered in this document.
+
```

Comment:
Add a note that you have to check the instructions for the version that you are going to install